kuma/core/decorators.py

```diff
@@ -136,3 +137,17 @@ def is_superuser(u):
 
 superuser_required = user_passes_test(is_superuser)
 #: A decorator to use for requiring a superuser
+
+
+def block_user_agents(view_func):
+    blockable_user_agents = []
+    if hasattr(settings, 'BLOCKABLE_USER_AGENTS'):
+        blockable_user_agents = settings.BLOCKABLE_USER_AGENTS
+
+    def agent_blocked_view(request, *args, **kwargs):
```

Comment:
Apple `functools.wraps` here like so: `@wraps(view_func, assigned=available_attrs(view_func))` where `available_attrs` is `django.utils.decorators.available_attrs`.